.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Excellent PR description!
Can we remove the WIP commits on this branch? (squash)